spec/Classes.tex

```diff
@@ -309,11 +309,14 @@ \subsection{Overriding Base Class Methods}
 dispatch in the event that a variable that has the base class type
 references an object that has the derived class type.
 
-The identical signature requires that the names, types, and order of
+The identical signature requires that the names, intents, types, and order of
```

Comment:
You've inherited this, but I think this either needs to say:

```Having the identical signature...```
or:
```"The identical signature" ...```

(where I prefer the former) because otherwise it sounds like "The identical signature" is a sentient entity requiring something from you.

I'd probably also say "means that" rather than "requires that" since this is defining things rather than prescribing.  (Or, I'd rewrite the previous paragraph to take more a prescription-based approach (e.g., "If a subclass wants to override a parent class method, that method must have an identical signature....  Having an identical signature requires...").  In contrast, the current text is using a style more like "If the signature happens to be the same..."